x-pack/plugins/lens/public/data_views_service/loader.test.ts

```diff
@@ -266,218 +254,4 @@ describe('loader', () => {
       expect(onError).not.toHaveBeenCalled();
     });
   });
-
-  describe('syncExistingFields', () => {
```

Comment:
The logic of fetching fields existence info was removed from Lens Redux store and moved into a hook `src/plugins/unified_field_list/public/hooks/use_existing_fields.ts`. Tested are added next to it.